Source/GUI/MemWatcher/MemWatchWidget.cpp

```diff
@@ -273,6 +273,42 @@ void MemWatchWidget::onMemWatchContextMenuRequested(const QPoint& pos)
   connect(copy, &QAction::triggered, this, [this] { copySelectedWatchesToClipBoard(); });
   contextMenu->addAction(copy);
 
+  if (index.isValid())
+  {
+    MemWatchEntry* const entry = m_watchModel->getEntryFromIndex(index);
+    if (entry->isBoundToPointer())
+    {
+      QMenu* const copyAddrSubmenu = contextMenu->addMenu(tr("Copy add&ress..."));
+      QAction* const copyPointer = new QAction(tr("Copy &base address..."), this);
+      const QString addrString{QString::number(entry->getConsoleAddress(), 16).toUpper()};
+      connect(copyPointer, &QAction::triggered, this,
+              [addrString] { QApplication::clipboard()->setText(addrString);
+      });
+      copyAddrSubmenu->addAction(copyPointer);
+      for (int i = 0; i < static_cast<int>(entry->getPointerLevel()); ++i)
+      {
+        std::string const strAddressOfPath = entry->getAddressStringForPointerLevel(i + 1);
```

Comment:
Is `strAddressOfPath` used?

Reply:
It's not. I forgot to remove it when I changed the method for checking if a pointer address is valid. Removing it.